common/snapshots.py

```diff
@@ -1241,11 +1242,27 @@ def takeSnapshot(self, sid, now, include_folders):
         has_errors = False  # TODO Fix inconsistent usage: Collects return value,
                             #      but errors are also checked via params[0]
 
-        if rsync_exit_code != 0:  # indicates an error
+        # dict of exit codes (as keys) that are treated as INFO only by BiT
+        # (not as ERROR). The values are message strings for the snapshot log.
+        rsync_non_error_exit_codes = {
+            0: _("Success"),
+            24: _("Partial transfer due to vanished source files (see 'man rsync')")
+        }
+
+        rsync_exit_code_msg = _("'rsync' ended with exit code {exit_code}".format(exit_code=rsync_exit_code))
+
+        if rsync_exit_code in rsync_non_error_exit_codes:
+            self.setTakeSnapshotMessage(0,
+                                        rsync_exit_code_msg + _(": {msg}".format(
+                                            msg=rsync_non_error_exit_codes[rsync_exit_code])))
+        elif rsync_exit_code > 0:  # indicates a rsync error
+            params[0] = True  # HACK to fix #489 (params[0] and has_errors should be merged)
+            self.setTakeSnapshotMessage(1,
+                                        rsync_exit_code_msg + _(": See 'man rsync' for more details"))
```

Comment:
Here I would exclude the `: ` from the string to translate. Translators on Weblate can overlook and miss it.

Modify this line

    rsync_exit_code_msg + _(": See 'man rsync' for more details"))

to this

    rsync_exit_code_msg + ": " + _("See 'man rsync' for more details"))